InMobiAdapter/src/main/java/com/chartboost/mediation/inmobiadapter/InMobiAdapter.kt

```diff
@@ -384,6 +369,45 @@ class InMobiAdapter : PartnerAdapter {
         }
     }
 
+    /**
+     * Build a [JSONObject] that will be passed to the InMobi SDK for GDPR during [setUp].
+     *
+     * @param gdprConsent A Boolean indicating whether GDPR consent is granted or not.
+     *
+     * @return a [JSONObject] object as to whether GDPR  .
+     */
+    private fun buildGdprJsonObject(gdprConsent: Boolean, context: Context): JSONObject {
+        return JSONObject().apply {
+            try {
+                put(InMobiSdk.IM_GDPR_CONSENT_AVAILABLE, gdprConsent)
+                put("gdpr", if (gdprApplies == true) "1" else "0")
+                getTCFString(context).takeIf { it.isNotEmpty() }?.let { tcfString ->
+                    put(InMobiSdk.IM_GDPR_CONSENT_IAB, tcfString)
+                } ?: run {
+                    PartnerLogController.log(CUSTOM, "TCFv2 String is empty or was not found.")
+                }
```

Comment:
Don't think we need the `run { }`

Reply:
Applied on: https://github.com/ChartBoost/chartboost-mediation-android-adapter-inmobi/pull/47/commits/2bbeff1a52ef6d8a91ac18acf85385666cffcd78